tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -23,14 +23,16 @@
 from tools_for_todoist.models.todoist import Todoist
 from tools_for_todoist.utils import ensure_datetime
 
-CALENDAR_EVENT_TODOIST_KEY = 'todoist_item_id'
 CALENDAR_EVENT_ID = 'calendar_event_id'
+CALENDAR_EVENT_TODOIST_KEY = 'todoist_item_id'
 
 
 def _todoist_id(calendar_event):
     todoist_id = calendar_event.get_private_info(CALENDAR_EVENT_TODOIST_KEY)
     if todoist_id is None:
         return None
+    if isinstance(todoist_id, int):
```

Comment:
this might be a miss on my part, when saving private info we should always convert it to string so it doesn't matter if it's just set or recently pulled from update, it should always be string and this method should return it as int or None